azurerm/internal/services/kusto/identity.go

```diff
@@ -22,44 +24,63 @@ func schemaIdentity() *schema.Schema {
 					Type:     schema.TypeString,
 					Required: true,
 					ValidateFunc: validation.StringInSlice([]string{
-						string(kusto.IdentityTypeNone),
 						string(kusto.IdentityTypeSystemAssigned),
+						string(kusto.IdentityTypeUserAssigned),
+						string(kusto.IdentityTypeSystemAssignedUserAssigned),
 					}, true),
 					DiffSuppressFunc: suppress.CaseDifference,
 				},
+
+				"identity_ids": {
+					Type:     schema.TypeSet,
+					Optional: true,
+					Elem: &schema.Schema{
```

Comment:
should this still not also be computed?

Reply:
@katbyte this is not a computed field. when the type is `SystemAssigned, UserAssigned`, users could set this property to pass the user managed identity. In this PR, I also add such an example

an example in VM: https://github.com/terraform-providers/terraform-provider-azurerm/blob/2598a62911c599278f06cf3fe9b57224e6bd3339/azurerm/internal/services/compute/virtual_machine.go#L70

Reply:
👍